src/commands/Changelog.ts

```diff
@@ -0,0 +1,318 @@
+import { ExpressionMetricMapping } from "../parser/helper/Model";
```

Comment:
This is not related to the marked line. But let's think about the location of this file and the name. Maybe it should be located somewhere else e.g. `src/commands/import-grammars/ExpressionChangelog.ts`

Reply:
I have created a import-grammars subdirectory and renamed the Changelog to NodeTypesChangelog.